pkg/driver/helmdriver.go

```diff
@@ -35,6 +35,10 @@ type helmDriver struct {
 
 var _ PackageDriver = (*helmDriver)(nil)
 
+// TODO: Temporarily hard coding but needs to be read from secrets
+var caFile = "/tmp/registry-mirror/CACERTCONTENT"
```

Comment:
It is now at /tmp/config/registry/ca.crt

Reply:
If there is any change to CACERTCONTENT, cat.crt won't be updated for something like 12 hours. It is not a good idea to copy that secret.